model/src/cmd/cmdmodel.rb.in

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/ruby
+
+# Copyright (C) 2021 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# We use 'dl' for Ruby <= 1.9.x and 'fiddle' for Ruby >= 2.0.x
+if RUBY_VERSION.split('.')[0] < '2'
+  require 'dl'
+  require 'dl/import'
+  include DL
+else
+  require 'fiddle'
+  require 'fiddle/import'
+  include Fiddle
+end
+
+require 'date'
+require 'optparse'
+
+# Constants.
+LIBRARY_NAME = '@model_library_location@'
+LIBRARY_VERSION = '@PROJECT_VERSION_FULL@'
+COMMON_OPTIONS =
+  "  -h [ --help ]              Print this help message.                   \n"\
+  "                                                                        \n"\
+
+COMMANDS = { 'model' =>
+  "Print information about models.\n\n"+
+  "                                                                        \n"\
+  "  ign model [options]                                                   \n"\
+  "                                                                        \n"\
+  "Available Options:                                                      \n"\
+  "  --list                     Get a list of the available models.        \n"\
+  "  -m [--model] arg           Select the model to be shown.              \n"\
+  "  -l [--link]  arg           Select a link to show its properties.      \n"\
+  "                             If no arg is passed all links are printed  \n"\
+  "                             Requires the -m option                     \n"\
+  "                                                                        \n"\
+  "  -j [--joint] arg           Select a joint to show its properties.     \n"\
+  "                             If no arg is passed all joints are printed \n"\
+  "                             Requires the -m option                     \n"\
+  "                                                                        \n"\
+  "                             E.g. to get information about the          \n"\
+  "                             box_link in the shapes world, run:         \n"\
+  "                             ign model -m box -l box_link               \n\n"+
```

Comment:
This example is talking about links, but it's technically under the joint option. I think it's worth having a separate example for each option.

Reply:
Addressed